tests/python/relax/frontend/test_onnx_frontend.py

```diff
@@ -1337,14 +1341,21 @@ def verify_pad_constant_value(constant_value):
     verify_pad_constant_value("")
 
 
-@pytest.mark.skip
-def test_split():
+@pytest.mark.parametrize(
```

Comment:
You can do two separate parametrizes (for each parameter) and it'll enumerate all the combinations.